config/config.go

```diff
@@ -1090,11 +1091,14 @@ func getSubnetConfigsFromDir(v *viper.Viper, subnetIDs []ids.ID) (map[ids.ID]sub
 
 	// reads subnet config files from a path and given subnetIDs and returns a map.
 	for _, subnetID := range subnetIDs {
+		config := getDefaultSubnetConfig(v)
```

Comment:
The comment added to `chains/manager.go` implies that we now require a default config is present for each `subnetID` that we're tracking. Do we need to guarantee that here as well? It seems that if `subnetConfigPath` is empty, then we'll exit before getting to this point and may not guarantee that?

Reply:
It seems like this actually can't cause a problem because there are two behaviors:
- use the default path where this will be non-empty and therefore not exit early
- use a non-default path, where it will then check that the directory exists, which will cause an error if the subnetConfigPath is in fact an empty string